build.gradle

```diff
@@ -93,6 +93,16 @@ subprojects {
           }
         }
 
+        def tcnative_suffix = "";
+        if (osdetector.classifier in ["linux-x86_64", "osx-x86_64", "windows-x86_64"]) {
```

Comment:
We discussed separately, but for those following along:
If you get the library without the classifier, you get all the Java classes, but none of the native components. So you can compile and run with the non-native transports, but if you try to use the native transport then you will get an error like the one in the commit message.